kubernetes/samples/scripts/create-fmw-infrastructure-domain/domain-home-on-pv/create-domain-inputs.yaml

```diff
@@ -149,3 +149,6 @@ rcuDatabaseURL: database:1521/service
 
 # The kubernetes secret containing the database credentials
 rcuCredentialsSecret: domain1-rcu-credentials
+
+# FMW Infrastcructure Domain Type. It can be JRF or RestrictedJRF
```

Comment:
"Infrastcructure" -> "Infrastructure".

"It can be JRF or RestrictedJRF" -> "Legal values are `JRF` or `RestrictedJRF`."